backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -184,6 +184,13 @@ public Buffer getPixels () {
 		return buffer;
 	}
 
+	/**
+	 * This method doesn't work with GWT.
+	 * @param flag
+	 */
+	static public void setFlipY(boolean flag) {

```

Comment:
Please use the libgdx formatter on the java code that you change/add.  Keep `public static` order for consistency also.